Documentation/Books/Manual/Foxx/Guides/DevelopmentMode.md

```diff
@@ -0,0 +1,75 @@
+Development mode
+================
+
+Development mode allows developers to make changes to deployed services
+in-place directly on the database server's file system without downloading
+and re-uploading the service bundle. This can help during rapid development
+of service prototypes or diagnosing complex problems.
+
+You can toggle development mode on and off using
+the [Foxx CLI](../../Programs/FoxxCLI/README.md),
+the [HTTP API](../../../HTTP/Foxx/Miscellaneous.html) or
+in the service settings tab of the web interface.
+Once activated the service's file system path will be shown in the info tab.
+
+In development mode the service's source files and manifest will be
+re-evaluated, and its setup script (if present) re-executed,
+every time a route of the service is accessed,
+effectively re-deploying the service on every request.
+Additionally, error responses generated by Foxx will include stacktraces and
+when viewed in a browser may include relevant sections of the service code
+that generated the error.
+
```

Comment:
We should explain somewhere here how to find the folder where the Foxx services are in. Is there a way to find out via arangosh? Otherwise, one would have to look at arangod.conf and possibly startup parameters specified on the command line that overwrite the default config.

Reply:
Fixed in https://github.com/arangodb/arangodb/pull/5842/commits/4d7b0b47c8e76d69a640bbe1691d928fb753317a